posthog/temporal/batch_exports/redshift_batch_export.py

```diff
@@ -211,7 +211,7 @@ def redshift_default_fields() -> list[BatchExportField]:
 
 
 def get_redshift_fields_from_record_schema(
-    record_schema: pa.Schema, known_super_columns: list[str], use_super: bool = False
+    record_schema: pa.Schema, known_super_columns: list[str], use_super: bool
```

Comment:
just double-checking that removing this default for `use_super` won't break anything?

Reply:
It's called only once, on this same module, so I don't see any issues here.

That being said, I prefer the version without a default as it forces the caller to be aware of whether they are using SUPER type or not, instead of relying on a default.